java/openjdk/Portfile

```
@@ -340,6 +280,18 @@ if {${subport} eq "openjdk8"} {
     # Remove after 2021-11-28
     PortGroup    obsolete 1.0
     replaced_by  openjdk11
+} elseif {[lsearch -exact {openjdk12 openjdk13 openjdk14} ${subport}] != -1} {
```

Must we use the old Tcl 8.4 `lsearch -exact` syntax? It is simpler to use the Tcl 8.5 `in` syntax.
```suggestion
} elseif {${subport} in [list openjdk12 openjdk13 openjdk14]} {
```

---

Thanks for the suggestion. Incorporated in e8d24eb8de.